### PR TITLE
Various small fixes to chat

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] update hint text. 
- [x] auto focus input. 
- [x] reorder inbox based on gregor. 
- [x] fix particpants being in reverse order
- [x] empty chat placeholder

### CODE REVIEW

File: shared/chat/conversation/container.js [hunk 1]
```diff
@@ -14,27 +15,46 @@ import type {Props} from '.'
 type OwnProps = {}
 type State = {
   sidePanelOpen: boolean,
+  inputText: string,
 }
 
 class ConversationContainer extends Component<void, Props, State> {
   state: State
+  _inputTextMap: {[key: string]: ?string}= {}
```
Comment (chrisnojima) [inline shared/chat/conversation/container.js]: keeping the input state in here for now so if you switch conversations its store but outside of redux. TODO make this into the route state

File: shared/chat/conversations-list/index.desktop.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const Row = ({onSelectConversation, selectedConversation, onNewChat, nowOverride
     backgroundColor: globalColors.darkBlue4,
     username: p.username,
     borderColor: rowBorderColor(idx, Math.min(2, participants.count()) - 1, hasUnread, isSelected),
-  })).toArray()
+  })).toArray().reverse()
```
Comment (chrisnojima) [inline shared/chat/conversations-list/index.desktop.js]: its recent first so we had it reversed

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by keybase-ci-visdiff** [comment]
The commits 40d4abd605bc...39324ade38ef introduce visual changes on linux. <details><summary>:mag_right: 8 changed</summary>

 * Changed: **Chat_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_List-Normal-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_List-Normal-39324ade38ef.png)  
   Chat_List-Normal comparison
 * Changed: **Chat_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_List-Empty-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_List-Empty-39324ade38ef.png)  
   Chat_List-Empty comparison
 * Changed: **Chat_Input-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Input-Normal-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Input-Normal-39324ade38ef.png)  
   Chat_Input-Normal comparison
 * Changed: **Chat_Input-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Input-Empty-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Input-Empty-39324ade38ef.png)  
   Chat_Input-Empty comparison
 * Changed: **Chat_Input-Emoji_Open** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Input-Emoji_Open-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Input-Emoji_Open-39324ade38ef.png)  
   Chat_Input-Emoji_Open comparison
 * Changed: **Chat_Conversations_List-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Conversations_List-Selected_Normal-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Conversations_List-Selected_Normal-39324ade38ef.png)  
   Chat_Conversations_List-Selected_Normal comparison
 * Changed: **Chat_Conversations_List-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Conversations_List-Selected_Muted-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Conversations_List-Selected_Muted-39324ade38ef.png)  
   Chat_Conversations_List-Selected_Muted comparison
 * Changed: **Chat_Conversations_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Conversations_List-Normal-40d4abd605bc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Conversations_List-Normal-39324ade38ef.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1481766970584-40d4abd605bc-39324ade38ef-linux/Chat_Conversations_List-Normal.png)
</summary>

File: shared/chat/conversation/container.js (hunk 1 above)
Comment (cjb) [inline shared/chat/conversation/container.js]: Hm, is the TODO something extra you want to do as part of this PR, or to happen after it's merged?  It won't be seen after this goes in, so it seems like it'd make more sense to leave the TODO as a comment in the source here, or create a JIRA ticket for it.

**Comment by chrisnojima** [comment]
done! DESKTOP-2392

**Pull request merged**